src/options.rs

```diff
@@ -225,7 +225,7 @@ fn cargo_dir() -> (String, PathBuf) {
     }
 }
 
-fn existing_dir_validator(label: &str, s: String) -> Result<(), String> {
+fn existing_dir_validator(label: &str, s: &str) -> Result<(), String> {
     fs::canonicalize(&s).map(|_| ()).map_err(|_| format!("{} directory \"{}\" not found", label, s))
```

Comment:
There's little difference here though, innit, seeing as the string will've gotten `drop()`ped by the end of the caller's line anyway, but good idiomaticity catch 👍 

(Although, does it work with `fs::canonicalise(s)` here? Seems superiour, innit?)

Reply:
I don't remember if I tried `fs::canonicalise(s)` but yeah looks like it should work.